plugins/source/aws/resources/services/eventbridge/event_buses_fetch.go

```diff
@@ -68,3 +69,31 @@ func resolveEventBridgeTags(ctx context.Context, meta schema.ClientMeta, resourc
 	}
 	return resource.Set(c.Name, client.TagsToMap(output.Tags))
 }
+
+func resolveArchiveArn(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

Comment:
I think you're cheating on the codegen a little here by putting some custom resolver code in another resource's file 😉

Reply:
Moved to `arn_resolvers.go` file